soc/arm/st_stm32/stm32f0/Kconfig.defconfig.stm32f072x8

```diff
@@ -0,0 +1,12 @@
+# ST Microelectronics STM32F072X8 MCU
+
+if SOC_STM32F072X8
+
+# STM HAL uses 'stm32f072xb' for both STM32F072XB and STM32F072X8 variants.
```

Comment:
In that case, please change `Kconfig.defconfig.stm32f072xb` file name to `Kconfig.defconfig.stm32f072xx` and factorize definitions.
Check [here](https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/st_stm32/stm32f1/Kconfig.defconfig.stm32f105xx) for an example.

Reply:
i combined both files into Kconfig.defconfig.stm32f072xx, as in the example :+1: